Makefile

```diff
@@ -24,3 +27,15 @@ $(BINDIR)/golangci-lint: $(BINDIR)
 
 $(BINDIR):
 	mkdir -p $(BINDIR)
+
+.PHONY: cross
+cross:
+	for os in $(CROSS_OSES); do \
+		for arch in $(CROSS_ARCHES); do \
+			echo "$$os/$$arch" | grep -qE $(OS_ARCH_SKIP) && continue; \
```

Comment:
Looks like the list is;

```
linux/amd64
linux/arm
linux/arm64
linux/ppc64le
linux/s390
freebsd/amd64
freebsd/arm
darwin/amd64
darwin/arm
darwin/arm64
windows/amd64
windows/arm
```

Wondering if we need:

- freebsd/arm
- darwin/arm (likely not, darwin won't be 32-bit)
- windows/arm (not sure; is windows on 32-bit arm a thing, or are they working on 64 bit only?)

If we remove those from the list;

```
linux/amd64
linux/arm
linux/arm64
linux/ppc64le
linux/s390
freebsd/amd64
darwin/amd64
darwin/arm64
windows/amd64
```

Wondering if it would be better to just define that list (easier read, and easier to maintain as well)

If would require splitting to GOOS and GOARCH then, so something like

```bash
GOARCH=${CROSS#*/}
GOOS=${CROSS%/*}
```

Or handle it with Make;

```makefile
.PHONY: linux/% darwin/% freebsd/% windows/%
linux/% darwin/% freebsd/% windows/%:
        echo GOOS=$(@D)
        echo GOARCH=$(@F)
```

Reply:
Indeed this looks cleaner. Done; PTAL